CODEOWNERS

```diff
@@ -1,2 +1,6 @@
+# The ipdx team is responsible for GitHub Management maintenance (at least through 2024)
+* @filecoin-project/ipdx
```

Comment:
But do they need immediate write access to this repo? IMO, any changes should go through our team (I'm mostly concerned about them already being a massive target).

Reply:
Not really, that only comes in handy when we have to resolve some issue with the project setup, but we can deal with that on a case-by-case basis.

Reply:
@Stebalien: 

> But do they need immediate write access to this repo?

To be clear, this isn't giving them permission (see discussion about the role of codeowners in thie repo [here](https://github.com/filecoin-project/github-mgmt/pull/61#issuecomment-2313319725)).

The whole of this file is affectively saying that ipdx is needed for any changes made outside https://github.com/filecoin-project/github-mgmt/blob/master/github/filecoin-project.yml .  I think that is good.  If anyone is making github-mgmt setup changes (which happens in the directories outside of `filecoin-project/github-mgmt/github`, then they should be engaged.)

(No further action planned by here, but feedback/ideas welcome.)

> I'm mostly concerned about them already being a massive target

Fair.  To be clear on this, the ipdx team as a whole doesn't have write access to this repo or any other I don't think.  @galargh individually is an org owner/admin and part of github-mgmt-stewards (according to this proposed PR), so I agree he's  a big target (as are the other couple of owners - which is why I'm trying to limit that set of people).

Potential action: remove @galargh from being a org owner or github-mgmt steward and replace him with someone else.  I didn't opt for that in this proposal/PR because I think the risk of his account being compromised is outweighed by if crap-hits-the-fan that we have someone observant, skilled, and knowledgable with github tools take action.  I'm happy to change course though.